.github/workflows/source_aws.yml

```diff
@@ -44,86 +34,46 @@ jobs:
       run:
         working-directory: ./plugins/source/aws
     steps:
-      - uses: actions/checkout@v3
-        with:
-          fetch-depth: 2
-      - name: Set up Go 1.x
+      - name: Set up Go
+        id:   set-up-go
         uses: actions/setup-go@v3
         with:
-          go-version-file: plugins/source/aws/go.mod
-          cache: true
+          go-version-file:       plugins/source/aws/go.mod
+          cache:                 true
           cache-dependency-path: plugins/source/aws/go.sum
-      - name: golangci-lint
-        uses: golangci/golangci-lint-action@v3
+      - name: Restore build cache
```

Comment:
We should first resolve https://github.com/cloudquery/cloudquery/pull/9656 before doing any caching related changes